Indicators/MovingAverageConvergenceDivergence.cs

```diff
@@ -80,7 +81,7 @@ public MovingAverageConvergenceDivergence(string name, int fastPeriod, int slowP
             Slow = type.AsIndicator(name + "_Slow", slowPeriod);
             Signal = type.AsIndicator(name + "_Signal", signalPeriod);
             Histogram = new Identity(name + "_Histogram");
-            WarmUpPeriod = signalPeriod;
+            WarmUpPeriod = Math.Max(fastPeriod, slowPeriod) + signalPeriod - 1;
```

Comment:
Should we throw if `fast >= slow`? It'll result in zero or negative `var macd = Fast.Current.Value - Slow.Current.Value;` then weirdness in `Histogram.Update(input.Time, macd - Signal.Current.Value);`

Reply:
Yes, good point. I'll add this tomorrow.